src/transform/RedLinks.js

```diff
@@ -12,9 +12,10 @@ const configureRedLinkTemplate = (span, anchor) => {
 /**
  * Finds red links in a document or document fragment.
  * @param {!(Document|DocumentFragment)} content Document or fragment in which to seek red links.
- * @return {!NodeList} Nodelist of zero or more red link anchors.
+ * @return {!HTMLAnchorElement[]} Array of zero or more red link anchors.
  */
-const redLinkAnchorsInContent = content => content.querySelectorAll('a.new')
+const redLinkAnchorsInContent =
+  content => Array.prototype.slice.call(content.querySelectorAll('a.new'))
```

Comment:
Note: we didn't use `Array.from` because it wasn't supported by all Android targets.